src/dataframe/dataframe.jl

```diff
@@ -102,6 +103,14 @@ function DataFrame(; kwargs...)
     return result
 end
 
+function DataFrame(pairs::Pair{Symbol}...)
```

Comment:
Yeah, but you can just use a generator rather than defining `pair` and calling `map`:
```julia
DataFrame((k => v for t in kwargs)...)
```